esmvaltool/recipes/hydrology/recipe_marrmot.yml

```diff
@@ -0,0 +1,72 @@
+# ESMValTool
+# recipe_marrmot.yml
+---
+documentation:
+  description: |
+    Recipe pre-processes ERA-Interim and ERA-5 data
+    for use in the MARRMoT hydrological model.
+
+    MARRMoT is the Modular Assessment of Rainfall-Runnoff Models Toolbox ...
+    https://www.geosci-model-dev.net/12/2463/2019/
+  authors: ['kalverla_peter', 'camphuijsen_jaro', 'alidoost_sarah']
+  projects: ['ewatercycle']
+  references: ['acknow_project']
+
+preprocessors:
+  extract_catchment: &catchment
+    extract_shape:
```

Comment:
This is needlessly complicated, as you only use the daily and hourly preprocessor funtions, so `extract_shape` can just go into the `daily` preprocessor and the `hourly` preprocessor can just inherit from that.

Reply:
thanks, done.